pom.xml

```diff
@@ -305,12 +313,12 @@
     </profiles>
     <distributionManagement>
         <snapshotRepository>
-            <id>nexus.gadevs</id>
-            <url>https://nexus.gadevs.ga/repository/maven-snapshots/</url>
+            <id>igssitelog-bindings-snapshot</id>
+            <url>s3://${gnss.artifactory.repo}/maven-snapshots/${project.artifactId}/${project.version}</url>
```

Comment:
Do you need so specify artifact id and project version? I think `mvn deploy` will make this directories for you. If you just end it with `.../maven-snapshots`, `mvn deploy` will use the group id, artifact id, and version to make a directory structure under maven-snapshots.

Reply:
Yes, you are right. I have removed the artifact id and project version from the path. Thanks.